app/src/main/java/sw19/moring03/paint/Fragments/ShapeChooserFragment.java

```diff
@@ -20,26 +20,26 @@ public Dialog onCreateDialog(@Nullable Bundle savedInstanceState) {
         AlertDialog.Builder builder = new AlertDialog.Builder(getActivity());
         builder.setTitle(R.string.shapeChooserFragment)
                 .setItems(R.array.shape_type, new DialogInterface.OnClickListener() {
-                    public void onClick(DialogInterface dialog, int which) {
-                        String[] types = getResources().getStringArray(R.array.shape_type);
+                            public void onClick(DialogInterface dialog, int which) {
```

Comment:
you just pressed tab in this whole function and therefore there are changes here, remove those tabs you haven't made changes in this file